metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -88,7 +88,7 @@ final class Doctor(
           onServer(server)
         case None =>
           scribe.warn(
-            "Unable to run doctor. To fix this problem enable -Dmetals.http=true"
+            "Unable to run doctor. Server has not yet started."
```

Comment:
If we change the above, then this message also becomes a bit more telling. Since it's no longer actually about havint `http=true`, but rather than the server isn't up yet. For example, this sometimes leads to confusion if a client has `httpEnabled`, when the first `metals-doctor-reload` happens right at startup, this falls in `None` showing the warning, even though the setting is true. This change will also avoid that and make the message clearer.